SETUP.md

```diff
@@ -25,6 +25,8 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 
 1. `rbenv rehash`
 
+1. `bundle config --local without staging test production levelbuilder`
```

Comment:
one alternative to consider would be to just run `bundle install --without staging test production levelbuilder` on the next line and remove this line. the `rake install` step a few lines later should set the appropriate bundle config:
* https://github.com/code-dot-org/code-dot-org/blob/2203800f49d92f55d18501a3e79fe4e3f2ccfc3f/lib/rake/install.rake#L46
https://github.com/code-dot-org/code-dot-org/blob/14946a976e9f27d51ff0cc8016810feefa28efe5/lib/cdo/rake_utils.rb#L154-L156

Reply:
I spent a little time before suggesting this direction exploring whether there was a clean trick for running the rake task BEFORE the first bundle install, but as @elijah guessed initially its a chicken and egg problem as currently configured

Reply:
`bundle install --without` works, but is listed as deprecated. I preferred that too, but I figure no point putting in a deprecated command even if its cleaner for now.